src/main/java/com/gotocompany/firehose/sink/grpc/client/GrpcClient.java

```diff
@@ -58,18 +59,20 @@ public DynamicMessage execute(byte[] logMessage, Headers headers) {
         Metadata metadata = buildMetadata(headers);
         try {
             Channel decoratedChannel = ClientInterceptors.intercept(managedChannel,
-                     MetadataUtils.newAttachHeadersInterceptor(metadata));
+                    MetadataUtils.newAttachHeadersInterceptor(metadata));
             firehoseInstrumentation.logDebug("Calling gRPC with metadata: {}", metadata.toString());
             byte[] response = ClientCalls.blockingUnaryCall(
                     decoratedChannel,
                     methodDescriptor,
                     decoratedDefaultCallOptions(),
                     logMessage);
-
             return stencilClient.parse(grpcSinkConfig.getSinkGrpcResponseSchemaProtoClass(), response);
-
         } catch (StatusRuntimeException sre) {
-            firehoseInstrumentation.logError("gRPC call failed with error message: {}", sre.getMessage());
+            if (sre.getStatus().getCode() == Status.Code.UNAVAILABLE) {
+                firehoseInstrumentation.logError("gRPC configurations are incorrect: {}", sre.getMessage());
+            } else {
+                firehoseInstrumentation.logError("gRPC call failed with error message: {}", sre.getMessage());
+            }
             firehoseInstrumentation.incrementCounter(Metrics.SINK_GRPC_ERROR_TOTAL,  "status=" + sre.getStatus().getCode());
         } catch (Exception e) {
             firehoseInstrumentation.logError("gRPC call failed with error message: {}", e.getMessage());
```

Comment:
do we need to include the test resource binary file which is newly added in the MR? If not, we can update .gitignore file.